No compatible version found: casperjs@'>=1.1.0 <1.2.0'

Since i updated to npm 2.0 i got this error when running `npm install`.

So wouldn't it be better to point to a specific beta (1.1.0-beta3) or to the last stable version (1.0.4)

```
npm ERR! Darwin 13.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.10.32
npm ERR! npm  v2.0.0
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: casperjs@'>=1.1.0 <1.2.0'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["1.1.0-beta2","1.1.0-beta3"]
npm ERR! notarget 
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
   },
   "dependencies": {
     "phantomjs": "~1.9.1",
-    "casperjs": "~1.1"
+    "casperjs": "1.1.0-beta3"
   },
   "devDependencies": {
     "grunt-contrib-jshint": "~0.1.1",

